### ISSUE TITLE
'ngx-dropzone' is an Angular component, then verify that it is part of this module

### ISSUE THREAD

**Issue by Jerome2606** [peterfreeman/ngx-dropzone#64]
I'm using Angular 5 and Angular cli 1.3.2
after using npm install ngx-dropzone and importing the module into mine, I'm having this issue

```
'ngx-dropzone' is not a known element:
1. If 'ngx-dropzone' is an Angular component, then verify that it is part of this module.
2. If 'ngx-dropzone' is a Web Component then add 'CUSTOM_ELEMENTS_SCHEMA' to the '@NgModule.schemas' of this component to suppress this message. ("
  [ERROR ->]<ngx-dropzone (change)="onSelect($event)">
```

in my module.ts:

```
import { NgxDropzoneModule} from 'ngx-dropzone';
@NgModule({
  imports: [
    ...
    NgxDropzoneModule,
    ...
```

in my drop-zone.component.html:

```
<ngx-dropzone (change)="onSelect($event)">
    <!-- <ngx-dropzone-label>{{UPLOAD_TITLE | translate}}</ngx-dropzone-label>
    <ngx-dropzone-preview *ngFor="let f of files" [removable]="true" (removed)="onRemove(f)">
        <ngx-dropzone-label>{{ f.name }} ({{ f.type }})</ngx-dropzone-label>
    </ngx-dropzone-preview> -->
  </ngx-dropzone>
```

in my drop-zone.component.ts:

```
onSelect(event) {
    console.log(event);
    this.files.push(...event.addedFiles);
  }
onRemove(event) {
    console.log(event);
    this.files.splice(this.files.indexOf(event), 1);
  }
```

I don't understand where to seek for to force my angular build to discover the module. I already restarted my computer.

Thanks

**Comment by peterfreeman** [comment]
Hi @Jerome2606,

I recommend you to try the following:
* Make sure the import of the module is correct. If you want to use it in a component in a submodule you created, try to import it into this submodule.
* Update your Angular and CLI version. You should at least use Angular 7 as this is the minimum dependency of the npm package.

Good luck! 🤓

**Comment by ShirinShirinak** [comment]
Peterfreeman is correct. I actually had the same issue and I tried everything for the entire day. It turned out I was importing things correctly BUT my app.module didn't know about my feature module (I was trying to use it in the feature module). 

When I did the import in the app.module.ts and was trying to use the library in the feature model, I was getting the same error as you. 

When I moved my import to my feature module and imported my feature module into the app.module (so app module would know about my other module), it fixed the issue.
